### ISSUE TITLE
Truly flexible network start time 

### ISSUE THREAD

**Issue by kennyshittu** [andela-Kshittu/conference-manager#3]
If we're to go by this requirement in the question: `The networking event can start no earlier than 4:00 and no later than 5:00.`  Then all networking events should start immediately after the last talk for the track ends, i.e if the last talk in a track ends 4:05 pm, then Networking event should start immediately (4:05 pm) and not wait till 5 pm before starting. 

However, if we're to go by the expected output in the question, then the above condition will not stand. Here's what track 2 looks like in the expected output 

Track 2:

09:00AM Communicating Over Distance 60min
10:00AM Rails Magic 60min
11:00AM Woah 30min
11:30AM Sit Down and Write 30min
12:00PM Lunch
01:00PM Accounting-Driven Development 45min
01:45PM Clojure Ate Scala (on my project) 45min
02:30PM A World Without HackerNews 30min
03:00PM Ruby on Rails Legacy App Maintenance 60min
`04:00PM Rails for Python Developers lightning`
`05:00PM Networking Event`

Notice how `Networking Event` started 05:00 pm even though `Rails for Python Developers` would have ended `4:05 PM` (since lightning = 5mins)

The current Implementation follows the flow of the later, i.e Networking event starts at 5:00 PM  and there's a follow-up PR here to in-case we want the condition in the question to stand.

https://github.com/andela-Kshittu/conference-manager/pull/2
